### PR TITLE
executor: speed up unit tests in executor package

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
speed up unit tests in executor package

### What is changed and how it works?
change TestRecoverTable from TestSuite1 to testRecoverTable

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by qw4990** [comment]
run-unit-test

File: executor/executor_test.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"github.com/pingcap/tidb/infoschema"
```
Comment (XuHuaiyu) [inline executor/executor_test.go]: move these lines down

File: executor/executor_test.go [hunk 2]
```diff
@@ -95,6 +95,7 @@ var _ = Suite(&testBypassSuite{})
 var _ = Suite(&testUpdateSuite{})
 var _ = Suite(&testOOMSuite{})
 var _ = Suite(&testPointGetSuite{})
+var _ = Suite(&testFastAnalyze{})
```
Comment (XuHuaiyu) [inline executor/executor_test.go]: Should this be testRecoverTable?

File: executor/executor_test.go (hunk 1 above)
Comment (qw4990) [reply executor/executor_test.go]: fixed

File: executor/executor_test.go (hunk 2 above)
Comment (qw4990) [reply executor/executor_test.go]: Yes... it has been fixed now.

**Comment by XuHuaiyu** [comment]
Can `WIP` be removed now?

**Comment by qw4990** [comment]
> Can `WIP` be removed now?

Yes, it is removed now.

**Pull request merged**